src/main/resources/application.properties

```diff
@@ -3,3 +3,12 @@ spring.application.name=GymProject
 trainerFilePath=classpath:trainerRepository.txt
 traineeFilePath=classpath:traineeRepository.txt
 trainingFilePath=classpath:trainingRepository.txt
+
+# MySQL DataSource settings
+
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username=postgres
+spring.datasource.password=postgres
```

Comment:
This password must be stored within environment variable and here you just reference this env var.